examples/components/CCFRAUD/upload_data/run.py

```diff
@@ -17,7 +17,7 @@
     3: [["South"], ["Midwest"], ["West", "Northeast"]],
     4: [["South"], ["West"], ["Midwest"], ["Northeast"]],
 }
-CATEGORICAL_PROPS = ["category", "region", "gender", "state", "job"]
+CATEGORICAL_PROPS = ["category", "region", "gender", "state"]
```

Comment:
This has nothing to do with VFL, right?

Reply:
if we add job in categorical props and then do one-hot encoding the number of columns will explode, moreover it does not seem to help the accuracy and or recall. So it's maybe better to leave it out.